ui/src/utils/utils.ts

```diff
@@ -58,45 +66,53 @@ export const getStatusClass = (status: string) => {
 		case "cancelled":
 			return "text-[#F5222D] bg-[#FFF1F0]"
 		case "running":
-			return "text-[#0958D9] bg-[#E6F4FF]"
+			return "text-primary-700 bg-primary-200"
 		case "scheduled":
-			return "text-[rgba(0,0,0,88)] bg-[#f0f0f0]"
+			return "text-[rgba(0,0,0,88)] bg-neutral-light"
 		default:
 			return "text-[rgba(0,0,0,88)] bg-transparent"
 	}
 }
 
 export const getConnectorInLowerCase = (connector: string) => {
 	const lowerConnector = connector.toLowerCase()
-	if (lowerConnector === "amazon s3" || lowerConnector === "s3") {
-		return "s3"
-	} else if (
-		lowerConnector === "apache iceberg" ||
-		lowerConnector === "iceberg"
-	) {
-		return "iceberg"
-	} else if (connector.toLowerCase() === "mongodb") {
-		return "mongodb"
-	} else if (connector.toLowerCase() === "postgres") {
-		return "postgres"
-	} else if (connector.toLowerCase() === "mysql") {
-		return "mysql"
-	} else if (connector.toLowerCase() === "oracle") {
-		return "oracle"
-	} else {
-		return connector.toLowerCase()
+
+	switch (lowerConnector) {
+		case DESTINATION_INTERNAL_TYPES.S3:
+		case DESTINATION_INTERNAL_TYPES.AMAZON_S3:
+			return DESTINATION_INTERNAL_TYPES.S3
+		case DESTINATION_INTERNAL_TYPES.ICEBERG:
+		case DESTINATION_INTERNAL_TYPES.APACHE_ICEBERG:
+			return DESTINATION_INTERNAL_TYPES.ICEBERG
+		case "mongodb":
+			return "mongodb"
+		case "postgres":
+			return "postgres"
+		case "mysql":
+			return "mysql"
+		case "oracle":
+			return "oracle"
+		default:
+			return lowerConnector
 	}
 }
 
 export const getCatalogInLowerCase = (catalog: string) => {
-	if (catalog === "AWS Glue" || catalog === "glue") {
-		return "glue"
-	} else if (catalog === "REST Catalog" || catalog === "rest") {
-		return "rest"
-	} else if (catalog === "JDBC Catalog" || catalog === "jdbc") {
-		return "jdbc"
-	} else if (catalog === "Hive Catalog" || catalog === "hive") {
-		return "hive"
+	switch (catalog) {
```

Comment:
are u sure about the cases? shouldnt for safety catalog.toLowercase() and then match?
```suggestion
	switch (catalog.toLowercase()) {
```